x-pack/filebeat/module/oracle/database_audit/test/ORCLCDB_j002_28264_20201007122838056263426565.aud.log-expected.json

```diff
@@ -0,0 +1,126 @@
+[
+    {
+        "@timestamp": "2020-10-07T16:28:38.000Z",
+        "client.address": "",
+        "client.domain": "",
+        "client.user.name": "",
+        "event.dataset": "oracle.database_audit",
+        "event.module": "oracle",
+        "event.timezone": "-02:00",
```

Comment:
I think the original event had a timestamp of Wed Oct  7 12:28:38 2020 -04:00, so shouldn't the event.timezone be -04:00?